src/Blorc.OpenIdConnect/Blorc.OpenIdConnect.csproj

```diff
@@ -11,6 +11,7 @@
     <RazorLangVersion>3.0</RazorLangVersion>
     <AddRazorSupportForMvc>true</AddRazorSupportForMvc>
     <AutoGenerateBindingRedirects>false</AutoGenerateBindingRedirects>
+    <Nullable>enable</Nullable>
```

Comment:
Nullable is already defined via the import at the bottom. The reason we have the import is that we can change this on all our repositories at once instead of having to patch csproj files.